src/main/proto/wfa/measurement/api/v2alpha/measurement_spec.proto

```diff
@@ -73,5 +73,6 @@ message MeasurementSpec {
     ReachAndFrequency reach_and_frequency = 4;
     Impression impression = 5;
     Duration duration = 6;
+    ReachAndFrequency direct_reach_and_frequency = 7;
```

Comment:
One question I wonder about is whether it is a better design to leave this API as it is and to have the Kingdom decide whether to use the MPC or to send a direct measurement request to the EDP.  What are your thoughts on the pros and cons of this?